vast/CMakeLists.txt

```diff
@@ -57,6 +57,16 @@ elseif (${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
     configure_file(
       \"${CMAKE_CURRENT_SOURCE_DIR}/services/launchd/com.tenzir.vast.plist.in\"
       \"\$ENV{HOME}/Library/LaunchAgents/com.tenzir.vast.plist\" @ONLY)")
+elseif (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+  # Not all Linux distros use systemd, but those we care about at the moment do.
+  # TODO: We don't consider CPACK_PACKAGE_INSTALL_PREFIX here, this could
+  # potentially be solved with a more involved CPack setup. See also the
+  # comment at the beginning of cmake/Package.cmake.
+  configure_file(
+    "${CMAKE_CURRENT_SOURCE_DIR}/services/systemd/vast.service.in"
+    "${CMAKE_CURRENT_BINARY_DIR}/vast.service" @ONLY)
+  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/vast.service"
+          DESTINATION "${CMAKE_INSTALL_LIBDIR}/systemd/system/")
```

Comment:
Now that we have this logic we should remove the `systemd` directory from the repository root and move the remaining contents over to `vast/services/systemd` as well.

This also requires changing one line in the Dockerfile in the repository root, which references `systemd/vast.yaml`.

Reply:
Why are we copying the service file into the docker container?

Reply:
We are not, we're just re-using the same `vast.yaml` file that we reference by default in the service file.

Reply:
We're doing this in a follow-up PR.